python/kfserving/README.md

```diff
@@ -1,33 +1,26 @@
-# KFServing
+## Development
 
-KFServing is a unit of model serving. KFServing's python libraries implement a standardized KFServer library that is extended by model serving frameworks such as XGBoost and PyTorch. It encapsulates data plane API definitions and storage retrieval for models.
+Use conda to create a new environment
 
-KFServing provides many funtionalities, including among others:
-
-* Registering a model and starting the server
-* Prediction Handler
-* Liveness Handler 
-* Metrics Handler 
+```
+conda create --name kfserving python=3.7
+conda activate kfserving
+```
 
-KFServing supports the following storage providers:
+Install package for development
 
-* Google Cloud Storage with a prefix: "gs://"
-* S3 Compatible Object Storage with a prefix "s3://"
-* Local filesystem with a prefix "/"
+```
+pip install -e .
+```
 
-To start the server locally on your machine for development needs, run the following command under this folder
+To install development requirements
 
 ```
-pip3 install -e .
+pip install -r dev_requirements.txt
```

Comment:
>dev_requirements [](start = 15, length = 16)

use a conda environment.yml instead? e.g. [https://github.com/fastai/fastai/blob/master/environment.yml](https://github.com/fastai/fastai/blob/master/environment.yml)

Reply:
changed to use the convention (also in the fastai setup.py) of using extra_requirements Dict in setup.py.